dialect_sqlite_tag.go

```diff
@@ -0,0 +1,8 @@
+//go:build sqlite
+// +build sqlite
```

Comment:
Isn't this still requiring the `sqlite` tag to work? Or do you mean that you can now import this library, without go-sqlite3 included (unless `-tags sqlite` is specified) and then just choose to do this in your own app?

Reply:
The second option. As long as the driver is loaded, Pop will work with it.

This change preserves backwards-compatible behavior, so you can include the `sqlite` tag to include the driver.